netcore/utility/UserPasswordAuthInit.java

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
Same as with `SecurityManager` implementation. Also, the use of `AuthInit` on the server side is deprecated in favor of `SecurityManager` so it really strikes me as unnecessary for a new client to validate with a deprecated feature of the server.

Reply:
Will look into this.  I just copied in the minimal amount to build the existing fake `SecurityManager` class without editing Java code.  We're probably just being lazy in the Java code and referring to something we shouldn't be.  May want to fix this in the main tree as well, same for the required `cmake` version.

Reply:
Don't think it makes sense right now to modify the Java code, since we'll ultimately be removing it and building from a shared directory.  Again, https://issues.apache.org/jira/browse/GEODE-9422 is tracking.